frontend/src/components/EmailConfirmation.tsx

```diff
@@ -0,0 +1,22 @@
+import { useState, useEffect } from 'react';
+import axios from "axios";
+import { useParams } from "react-router-dom";
+
+function EmailConfirmation() {
+    const { id } = useParams<{ id: string }>();
+    const [confirmed, setConfirmed] = useState({confirmed: false})
```

Comment:
You don't need to use object for the state here.

You could just say: `useState(false)`, now `confirmed` variable will be `true` or `false`.

Otherwise you would have to write: `confirmed.confirm` to get the boolean value.

Reply:
Also, do you use `confirmed` variable anywhere?

Reply:
No, I don't.

Reply:
Then it's probably a good idea to remove it r comment it out if it might come in handy later.